azure-pipelines/localization.yml

```diff
@@ -0,0 +1,22 @@
+trigger:
+- main
+pr: none
+
+pool:
+  vmImage: windows-latest
+
+steps:
+- task: OneLocBuild@2
+  inputs:
+    locProj: 'LocProject.json'
+    outDir: '$(Build.ArtifactStagingDirectory)'
+    isCreatePrSelected: true
+    isShouldReusePrSelected: true
+    repoType: 'gitHub'
+    prSourceBranchPrefix: 'locfiles'
+    isAutoCompletePrSelected: false
+    isUseLfLineEndingsSelected: true
+    gitHubPatVariable: '$(GitHubPat)'
+    packageSourceAuth: patAuth
+    xLocConsoleLoggingLevel: debug
+    patVariable: '$(PackageSourcePat)'
```

Comment:
Can these secrets use the same mechanism used in the signing pipeline for releases, or is the "deploy key" thing there inapplicable here? (I want to make sure we aren't building team shared infrastructure that depends on an individual's credentials etc.)

https://github.com/microsoft/vcpkg-tool/blob/aee79d78ec25ad162197a1f7c6f5a394a9741837/azure-pipelines/signing.yml#L476-L495

Reply:
I don't think it can; I don't know the best way to do this, but we need specifically a PAT for a user that has write access to the repository.